sdk/spring/spring-cloud-azure-appconfiguration-config-web/CHANGELOG.md

```diff
@@ -18,6 +18,13 @@ Please refer to [spring/CHANGELOG.md](https://github.com/Azure/azure-sdk-for-jav
 
 Please refer to [spring/CHANGELOG.md](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/CHANGELOG.md#4120-2023-10-23) for more details.
 
+## 4.12.0-beta.1 (2023-09-11)
```

Comment:
The time should be "2023-11-09"?

Reply:
That was when the beta was released. This PR will go out for 4.12.0

Reply:
I see. Thanks